pandas/core/indexes/multi.py

```diff
@@ -545,7 +545,16 @@ def from_tuples(
         elif is_iterator(tuples):
             tuples = list(tuples)
         tuples = cast(Collection[Tuple[Hashable, ...]], tuples)
-
+        if len(tuples) and all((isinstance(e, tuple) and not e) for e in tuples):
```

Comment:
add a blank line then a comment that this is handling the empty tuple case

Reply:
Done